PerDiem/PerDiem/Controllers/TransactionsViewController.m

```diff
@@ -204,10 +224,14 @@ - (NSArray *)rightButtons
 
 - (void)swipeableTableViewCell:(SWTableViewCell *)cell didTriggerRightUtilityButtonWithIndex:(NSInteger)index {
     NSIndexPath *indexPath = [self.tableView indexPathForCell:cell];
+    NSInteger indexPathNumber = indexPath.row;
+    if (self.budget || self.period) {
```

Comment:
this condition is to fix the indexPathNumber. I don't think it's related to the loader. is it?